### DISCUSSION TITLE
Aggregate emails

### DISCUSSION THREAD

**Discussion by valadas** [community/community#14059] [Code Security]
I have a lot of repositories with a lot of technology debt, I set up dependabot on almost every repository and the tool is great overall. I did set it up for monthly alerts everywhere and those show up on the 1st of every month. With the people I know, we now call the 1st of the month the "dependabot" day, we all get just sooooo many emails.

It would be nice to be able to aggregate those many many emails somehow, maybe one per day, or one per repo or even to turn off emails completelly.

**Comment by david-poindexter** [comment]
<img src="https://media3.giphy.com/media/QMZpnb79N5BN0wsSM3/giphy.gif"/>
